src/frontends/onnx/tests/models/multinomial.prototxt

```diff
@@ -0,0 +1,49 @@
+ir_version: 7
+graph {
+  node {
+    input: "input"
+    output: "output"
+    op_type: "Multinomial"
+    attribute {
+      name: "sample_size"
```

Comment:
Here we have to add any seed != 0, otherwise tests may fail since it's a random sampling